source/_data/team.yml

```
@@ -0,0 +1,48 @@
+---
+- name: "Yevhen Kabanov"
+  image: "/img/cheatfate.jpg"
+  statusLink: "https://join.status.im/user/0x04f05d31956e9c918c35c30defe1ab69bc693d27eb3f79d619c3ba4f9f044b2349a79fb38e3a614ce9af1286910aa19bf24b53c18d48e10657cbf615a380928f6e"
+  statusName: "Fluid Stimulating Puma"
+  gitHubLink: "https://github.com/cheatfate"
+
+- name: "Jacek Sieka"
+  image: "/img/jacek.jpg"
+  statusLink: "https://join.status.im/user/0x049bc24b377c7e97bb94ab993e48fcbfe38f96aad43a42418d7f33f5a99f6f89e2af91875bfe045205771f28c93be50380cdfe5772201fde7019d57f7ead9af9b5"
+  statusName: "Piercing Double Honeyeater"
+  gitHubLink: "https://github.com/arnetheduck"
+
+- name: "Mamy Ratsimbazafy"
+  image: "/img/mamy.jpg"
+  statusLink: "https://join.status.im/user/0x04d01118c9cbcf4f9d326c1185bff110d36c96890a37cc47e7ce91e8b53ccc088d9df8c828c0669fb97831502113c1195bb17f1ea25fbccffee6a7fb9e30782a82"
+  statusName: "Damp Soulful Allensbigearedbat"
+  gitHubLink: "https://github.com/mratsim"
+
+- name: "Zahary Karadjov"
+  image: "/img/zahary.jpg"
+  statusLink: "https://join.status.im/user/0x049004fb151893f7112d754d92c650d6d35f8151ec0eff1c148812f8741474f7f8906436e782432680b40e96a2549c03d3dd7801c50928c2bf15d56f6d7d2e1246"
+  statusName: "Useless Gainsboro Serpent"
```

```suggestion
  statusName: "Nutritious Judicious Flicker"
```

That's my new name since v1.0

---

Nice thanks for the update.